### PR TITLE
Add Press page

### PR DESCRIPTION
**Pull request by KennethAshley**
##### Description
This integrates the press page with Netlify. Basically we took the blog infrastructure and tweaked it to work with the press page. 

##### Checklist

- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [ ] Tests coverage hasn't decreased

#### Todo
- [x] Tests
- [ ] Test with Netlify

##### Affected core subsystem(s)
UI, UX, Tests

##### Refers/Fixes
Fixes: #93 

# Example
screencapture-localhost-3000-press-2018-07-22-17_03_05

### CODE REVIEW

**Comment by KennethAshley** [comment]
@eswarasai depending on how similar we want the press pages to be to the blog pages we can reuse a few things like the `postPreviews` and even the `BlogPosts` components.

**Comment by KennethAshley** [comment]
@eswarasai @pelsasser check out the example and let me know what you think!

**Comment by KennethAshley** [comment]
@eswarasai @pelsasser What do you guys think? we do need to test this with Netlify.

**Comment by pelsasser** [comment]
@KennethAshley - just took a look at the screenshot. 

I pulled up the current https://marketprotocol.io/press/
against your branch  / screenshot mock.

A few things.

- I don't think we need categories for the press entries
- perhaps the hero / header should just read `Press`
- The `Continue Reading` link should direct to the actual article, maybe we stick with `Read More` similar to the existing page. 
- sharing icons are cool addition!

I will @eswarasai do the code review, but those are just the visual elements I noted. 

Thanks as always @KennethAshley !

**Comment by KennethAshley** [comment]
@pelsasser if we are linking to the actual article think we should even have a show page for each individual press item?

**Comment by pelsasser** [comment]
what do you mean by a show page? I think the main /press page will be the only page we host, the links will all take people to external sites.

**Comment by KennethAshley** [comment]
@pelsasser ah let me clarify. So currently the press page is similar to the blog page as in there is a list of posts and when we click on one post it'll go to that page to show all the content. but on this new press page when clicking on the post we want to link to that article and remove the functionality of that individual post having its own page right? 

If that is the case i guess we might want to ensure that when entering data we write a small blurb instead of long text so that we can show everything on the page that list all the posts. What do you think?

**Comment by pelsasser** [comment]
> @pelsasser ah let me clarify. So currently the press page is similar to the blog page as in there is a list of posts and when we click on one post it'll go to that page to show all the content. but on this new press page when clicking on the post we want to link to that article and remove the functionality of that individual post having its own page right?

@KennethAshley - yes!

This sounds like the right approach to me.

**Comment by eswarasai** [comment]
@KennethAshley -- Please find below feedback on the PR :
- Remove `src/.DS_Store`. Isn't this already in `.gitignore`? If not, please do add it
- `Press` should be in between `Partners` and `Blog` on the Header Menu
- There are 3 different `SocialLinks.tsx` components in the codebase in which 2 are duplicates. Is it possible to refactor the one in `components` folder to be able to be imported within `Blog` as well as `Press` just by passing few `props`?

@pelsasser -- In order to test the Netlify integration, we need to create the `press` collection. I don't have admin access in order to do that. Let me know if this is something you can take care of, so that we can do end-to-end testing.

@KennethAshley -- I'm assuming the above discussed UI changes in the comments are still pending or you're completely done with the UI piece?

**Comment by pelsasser** [comment]
@eswarasai - no admin needed there, we just create the collection on `config.yml` and you can login via your github creds - we will have to push to prod for it to be able to be tested I am afraid, but shouldn't be too big of a deal. 

Note to self: do not push to `master` until we fix bug with blog posts

**Comment by KennethAshley** [comment]
@eswarasai I can reuse a lot of the same things but i think there will need to be significant refactoring which is fine as i've already started doing this.

**Comment by eswarasai** [comment]
@KennethAshley -- If you think the refactoring takes a lot of time, you can hold off on that and wrap up this PR first. We still need to test the Netlify integration. Once this PR is merged, you can focus on refactoring things. Sounds good?

**Comment by KennethAshley** [comment]
@eswarasai it might take a day or two refactoring to get everything working. the press page will have a slightly different layout with no sidebar, and the post wont have their own page. So not drastic but will require some work. I might be able to have it done in two days.

**Comment by eswarasai** [comment]
@KennethAshley -- There's no need to refactor everything i.e., `Post`, `PostPreview` components etc. It'll be difficult if we do the refactor to change the layout if we'd like to in the future. I'm just talking about refactoring the `SocialLinks` as I don't see it changing much. Makes sense?

**Comment by KennethAshley** [comment]
@eswarasai ah i see! i misread what you were saying. I can get `SocialLinks` refactored soon.

**Comment by KennethAshley** [comment]
@eswarasai we should be good to go but i do want to work on test coverage.

**Comment by KennethAshley** [comment]
@eswarasai Yea i think it'll be a couple days to get the cover up. Ill ping you when im done!

**Comment by KennethAshley** [comment]
@eswarasai phew! a bit challenging on some areas but i got coverage up!

File: public/admin/config.yml [hunk 1]
```diff
@@ -23,3 +23,18 @@ collections:
...
+  - name: "press" # Used in routes, e.g., /admin/collections/blog
+    label: "Post" # Used in the UI
+    folder: "src/pressPosts" # The path to the folder where the documents are stored
+    create: true # Allow users to create new documents in this collection
+    slug: "{{year}}-{{month}}-{{day}}-{{slug}}" # Filename template, e.g., YYYY-MM-DD-title.md
+    fields: # The fields for each document, usually in front matter.
+            # Remove any that aren't needed for posts
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Author", name: "author", widget: "string"}
+      - {label: "Category", name: "category", widget: "select", options: ["About", "Trading", "Dex", "Partnerships", "Development", "Team", "Cryptocurrency"]}
```
Comment (eswarasai) [inline public/admin/config.yml]: @KennethAshley -- We don't need categories for `Press`. Can you please remove this from config as well as from the UI in the Press Post? Thanks!

**Comment by eswarasai** [comment]
@KennethAshley -- Readers also enjoyed section under Blog Post is messed up as you can see in the screenshot. Can you please put in a fix for this? Thanks!

screen shot 2018-08-07 at 11 12 03 am

**Comment by rojosnow** [comment]
@KennethAshley Do you have an update on this issue and @eswarasai comments?

File: public/admin/config.yml (hunk 1 above)
Comment (KennethAshley) [reply public/admin/config.yml]: Let me fix this!

**Comment by KennethAshley** [comment]
@rojosnow looking at this now.

**Comment by rojosnow** [comment]
@KennethAshley Thanks! @eswarasai All green, ready to go?

**Pull request merged**